tests/test_requests.py

```diff
@@ -1466,6 +1466,31 @@ def test_environment_comes_after_session(self, httpbin):
         with pytest.raises(KeyError):
             proxies['http']
 
+    def test_merge_environment_settings_verify(self):
+        """Assert CA environment settings are merged as expected when missing"""
+        session = requests.Session()
+
+        old_curl_bundle = os.environ.get('CURL_CA_BUNDLE')
+        old_requests_bundle = os.environ.get('REQUESTS_CA_BUNDLE')
+        if old_curl_bundle:
+            del os.environ['CURL_CA_BUNDLE']
+        if old_requests_bundle:
+            del os.environ['REQUESTS_CA_BUNDLE']
```

Comment:
Let's save ourself a bunch of time and code maintenance and just use [py.test's monkeypatch fixture](http://doc.pytest.org/en/latest/monkeypatch.html#method-reference-of-the-monkeypatch-fixture).